policyengine_canada/variables/gov/provinces/sk/tax/income/credits/spouse_or_common_law_partner_amount/sk_spouse_or_common_law_partner_credit_eligible.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_canada.model_api import *
+
+
+class sk_spouse_or_common_law_partner_credit_eligible(Variable):
+    value_type = float
+    entity = Household
+    label = "Eligible for the Saskatchewan spouse or common law partner credit"
+    definition_period = YEAR
+    reference = (
+        "https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1sk/td1sk-22e.pdf#page=1",
+        "https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1sk/td1sk-23e.pdf#page=1",
+        "https://publications.saskatchewan.ca/api/v1/products/583/formats/806/download#page=12",
+    )
+    defined_for = ProvinceCode.SK
+
+    def formula(household, period, parameters):
+        live_with_spouse = household("cohabitating_spouses", period)
+        person = household.members
+        head = person("is_head", period)
+
+        return household.any(head) * live_with_spouse
```

Comment:
Hi Max, thank you for reviewing! Yes, all households have a head. I write in this way because the entity of "sk_spouse_or_common_law_partner_credit_eligible" is "household" but the entity of "is_head" is "person".

Reply:
I understand but what is the purpose of multiplying by true?